protos/Makefile

```diff
@@ -1,23 +1,54 @@
 #
-#  Copyright 2018-2021 Dgraph Labs, Inc.
+# Copyright 2023 Dgraph Labs, Inc. and Contributors
 #
-#  This file is available under the Apache License, Version 2.0,
-#  with the Commons Clause restriction.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
 #
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+GOGO_PATH   := github.com/gogo/protobuf
 
-PROTO_PATH := ${GOPATH}/src:.
-PROTO_PATH := ${PROTO_PATH}:${GOPATH}/src/github.com/gogo/protobuf
+TMPDIR     := $(shell mktemp -d)
+PROTO_PATH := ${TMPDIR}/src:.
```

Comment:
what is the diff between this protos Makefile and the one in the dgraph repo?  can we keep them the same?

Reply:
They can't be exactly the same because the proto there depends on this proto.